src/pages/SupplierRequisitionPage.js

```diff
@@ -49,9 +53,17 @@ export class SupplierRequisitionPage extends React.Component {
     };
   }
 
+  componentDidMount = () => {
+    const { requisition, settings } = this.props;
+    if (requisition.program) {
+      const tags = settings.get(SETTINGS_KEYS.THIS_STORE_TAGS);
```

Comment:
I've just noticed that thresholdMOS is not in mobile schema, and doesn't sync to mobile.
I think this needs to happen. The only reason you need to find orderType object here is to get thresholdMOS. It should be assigned as requisition is created.
If orderTypes change, existing requisitions should not be affected.

Reply:
as per above, we don't need componentDidMount method.

Reply:
- Added `thresholdMOS` 5e15d9b71efa180f31d83a288659732de7d25e40
- Removed `componentDidMount` b582583b51aa8e56582f4c80a5ede306971861b0